packages/components/src/utils/token/TokenService.ts

```diff
@@ -0,0 +1,24 @@
+import { LocalStorage } from '../../localStorage';
+
+export class TokenService {
+  static TOKEN_PARAM = 'token';
+
+  constructor(private tokenKey: string) {}
+
+  async getToken(): Promise<string> {
```

Comment:
dont return Promise, through out this file

Reply:
error TS1064: The return type of an async function or method must be the global Promise<T> type.

Reply:
tscofig badalni hogi, isko hatane ke liye